docs/appendices/release-notes/5.6.0.rst

```diff
@@ -87,6 +87,8 @@ Performance and Resilience Improvements
 
 - Reduced the amount of disk reads necessary for ``ANALYZE`` operations.
 
+- Added support for filter push down beneath a view over multiple joins.
```

Comment:
I think something like "Add support in the optimizer to move filters beneath left/right joins." would fit better.

Reply:
True, how about something like "Improved filter push-down for left/right outer joins when joins are nested." maybe even with an example?